### PR TITLE
Bump enterprise debian Version

### PR DESCRIPTION
**Pull request by onurctirtir**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,13 @@
+citus-enterprise (9.2.4.citus-1) stable; urgency=low
+
+  * Fixes build errors on EL/OL 6 based distros
+
+  * Fixes a bug that caused maintenance daemon to fail on standby nodes
+
+  * Disallows distributed function creation when replication_model is statement
+
+ -- Burak Velioglu <velioglub@citusdata.com>  Tue, 31 Mar 2020 8:7:33 +0000
```
Comment (onurctirtir) [inline debian/changelog]:
```suggestion
 -- Onur Tirtir <Onur.Tirtir@microsoft.com>  Tue, 31 Mar 2020 8:7:33 +0000
```

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,13 @@
+citus-enterprise (9.2.4.citus-1) stable; urgency=low
+
+  * Fixes build errors on EL/OL 6 based distros
+
+  * Fixes a bug that caused maintenance daemon to fail on standby nodes
+
+  * Disallows distributed function creation when replication_model is statement
+
+ -- Onur Tirtir <Onur.Tirtir@microsoft.com>  Tue, 31 Mar 2020 8:7:33 +0000
```
Comment (onurctirtir) [inline debian/changelog]:
```suggestion
 -- Onur Tirtir <Onur.Tirtir@microsoft.com>  Tue, 31 Mar 2020 08:07:33 +0000
```

**Review by hanefi — CHANGES_REQUESTED** [review]
The changelog items appear to be wrong, can we manually fix them please?

File: debian/changelog [hunk 3]
```diff
@@ -1,3 +1,13 @@
+citus-enterprise (9.2.4.citus-1) stable; urgency=low
+
+  * Fixes build errors on EL/OL 6 based distros
+
+  * Fixes a bug that caused maintenance daemon to fail on standby nodes
+
+  * Disallows distributed function creation when replication_model is statement
```
Comment (onurctirtir) [inline debian/changelog]:
```suggestion
  * Fixes a release problem in 9.2.3
```

**Review by hanefi — APPROVED** [review]
:shipit:

**Pull request merged**